### PR TITLE
Check account balances before we allow money transfers #183691063

### PR DESCRIPTION
**Pull request by ericanderson91**

### CODE REVIEW

File: bxfinance/src/pages/pay-transfer/accounts-transfer.js [hunk 1]
```diff
@@ -87,37 +89,41 @@ class AccountsTransfer extends React.Component {
...
         if (!this.state.xfrAmount || this.state.xfrAmount <= 0) {
             this.setState({
                 accountError: 'Please enter a valid transfer amount.',
-                transferStarted: false
             });
             return;
         }
 
+        const fromAccount = this.state.accts.find(({acctNumber}) => acctNumber === this.state.fromValue);
+        console.log(fromAccount, +this.state.xfrAmount);
```
Comment (michaelsanchez-pingidentity) [inline bxfinance/src/pages/pay-transfer/accounts-transfer.js:114]: make sure to remove all console.logs before we push. or any other troubleshooting code.

**Review by michaelsanchez-pingidentity — CHANGES_REQUESTED** [review]
Good stuff. Just one catch on leftover logging.

File: bxfinance/src/pages/pay-transfer/accounts-transfer.js (hunk 1 above)
Comment (ericanderson91) [reply bxfinance/src/pages/pay-transfer/accounts-transfer.js:114]: Good catch

**Review by michaelsanchez-pingidentity — APPROVED** [review]
Reviewed.

**Pull request merged**